actions/transactions/new.go

```diff
@@ -61,3 +61,19 @@ func (a *Action) newTransaction(c *gin.Context) {
 	contract := mappings.MapToDraftTransactionContract(transaction)
 	c.JSON(http.StatusCreated, contract)
 }
+
```

Comment:
We should do one of those:
- Mark swagger above that this endpoint is deprecated and add some notice in description where to find new one.
- Use single method and comment @DeprecatedRouter to mark old route as deprecated (and adjust summary and description to have `draft` word in it)

Reply:
@dorzepowski `DeprecatedRouter` removes endpoint from the page so it's not a way to go with.

Reply:
Fixed. @DeprecatedRouter have some limitations which makes it useless in this context.